net.redeclipse.RedEclipse.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "net.redeclipse.RedEclipse",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        /* Red Eclipse does not respect XDG base directories */
+        "--persist=.redeclipse"
+    ],
+    "command": "redeclipse",
+    "rename-appdata-file": "redeclipse.appdata.xml",
+    "rename-desktop-file": "redeclipse.desktop",
+    "rename-icon": "redeclipse",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
+            "name": "redeclipse",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/red-eclipse/base/releases/download/v1.6.0/redeclipse_1.6.0_nix.tar.bz2",
+                    "sha256": "48a947e858587116b7d177cf18148d0d9e40b1c5b481e2c955baed68cffc2849"
+                },
+                {
+                    "type": "file",
+                    "path": "redeclipse.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "redeclipse.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "redeclipse.sh"
+                }
+            ],
+            "build-commands": [
+                "cd src; cmake; make",
```

Comment:
`no-make-install` + cmake-ninja buildsystem?

Reply:
Nope, I didn't contact upstream yet. I am however planning to do it.

RedEclipse already [has](https://github.com/red-eclipse/base/blob/master/src/install/nix/redeclipse.appdata.xml.am) an AppData file, but it is only a template that packagers can use, it is not in the Makefile and does not seem to be updated a lot. I have at least [submitted](https://github.com/red-eclipse/base/pull/824/commits/f81a76a66fe828ed43420cf3c4d6b8d278594a96) the OARS metadata, but I think it's better to use my own modified AppData file for now.

Same with the desktop file, it [exists](https://github.com/red-eclipse/base/blob/master/src/install/nix/redeclipse.desktop.am) as a template, but currently has an issue with the Keywords having bad separator which I [fixed](https://github.com/red-eclipse/base/pull/824/commits/cebec9e52877eebc17fd2e1d11ef9c1a047498aa) upstream, but it won't be fixed in the current version. I am also not sure, how to easily replace the "@APPNAME@" strings. So I am using my own desktop file for now.

Reply:
The reason why I used the _simple_ buildsystem is that the CMake files are not in the main directory, but in the "src" directory. Is there any way to specify the build directory?

Reply:
@jcaesar Thanks!